xregexp/xregexp-tests.ts

```diff
@@ -1,8 +1,7 @@
 /// <reference path="xregexp.d.ts" />
 
-import X = require('xregexp');
-import XRegExp = X.XRegExp;
-import TokenOpts = X.TokenOpts;
+import XRegExp = xregexp.OuterXRegExp;
```

Comment:
I think this could break existing clients. Some clients seem to reference "X.XRegExp". The "OuterXRegExp" is not public and if at all possible, it would be good if X.XRegExp could still work.